eval/src/vespa/eval/tensor/dense/dense_tensor_value_builder.h

```diff
@@ -0,0 +1,32 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "dense_tensor.h"
+
+namespace vespalib::tensor {
+
+/**
+ * A builder for DenseTensor objects
+ **/
+template<typename T>
+class DenseTensorValueBuilder : public eval::ValueBuilder<T>
+{
+private:
+    eval::ValueType _type;
+    std::vector<T> _cells;
+public:
+    DenseTensorValueBuilder(const eval::ValueType &type, size_t num_mapped_in,
```

Comment:
I suggest just dropping unneeded parameters; num_mapped_in and expected_subspaces